src/papers/smart-token-paper.md

```diff
@@ -97,13 +97,15 @@ In essence, the trajectory of Web 2.0 innovation is not solely constrained by th
 
-Rather than viewing the centralisation of value as the *cause* of the Web's current evolutionary obstacles, we argue that it is, in fact, a *consequence* of the Web's evolution. The transition from Web 1.0 to Web 2.0 was not primarily driven by a pursuit of centralised value, but by the demand for more dynamic, application-oriented experiences. Therefore, it is not logical to assume that the transition to Web 3.0 should be defined by a reversal of this trend.
+Rather than viewing the centralisation of value as the *cause* of the Web's current evolutionary obstacles, we argue that it is, in fact, a *consequence* of the Web's evolution. The historical shift from Web 1.0 to Web 2.0 was not predominantly driven by a collective quest for value but was propelled by the demand for more interactive, application-centric experiences. Consequently, it is reductive to presume that the evolution to Web 3.0 should be predicated on a reversal of this trajectory.
 
-Furthermore, the characterization of Web 3.0 as an 'Internet of Value' may be more reflective of a countercultural movement against the early elitism Web 2.0, spurred by social and economic disparities, rather than an evolutionary process of the Web. Not everyone who transitioned from prior-generation Web to Web 2.0 was pursuing value, and not everyone who migrates from Web 2.0 to the next-generation web will necessarily be pursuing value either. While the value investing community is growing, it still represents a small section of web users and is likely to remain so in the future.
+The portrayal of Web 3.0 as an 'Internet of Value' is more aptly interpreted as a countercultural movement against the perceived elitism of Web 2.0, spurred by socio-economic disparity, rather than an evolutionary process of the Web. It's not Web has "come of age" to be a finance application, but a coming-of-age user base increasingly sought financial autonomy from a platform that has historically been perceived as a panacea.
+
+Not everyone who transitioned from prior-generation Web to Web 2.0 was pursuing value, and not everyone who migrates from Web 2.0 to the next-generation web will necessarily be pursuing value either. While the value investing community is growing, it still represents a small section of web users and is likely to remain so in the future. 
 
 Instead of defining Web 3.0 as an 'Internet of Value,' this paper proposes an alternative vision: the 'Next-Generation Web.' This Next-Generation Web shifts focus from the distribution of value to the decentralisation of Trust Anchors, attacking the root cause of centralisation while continuing to progress in the demand-driven direction that has been driving the Web 1.0 to 2.0 upgrade. This perspective views the Next-Generation Web as a platform that fosters widespread innovation and enables more profound integrations by allowing anyone to develop and maintain Trust Anchors.
```

Comment:
Suggestion;

Instead of defining Web 3.0 as an 'User-centric Internet of Value,' this paper proposes an alternative vision: the 'User-centric Next-Generation Web.'